README.md

```diff
@@ -17,7 +17,7 @@ import fluent from 'fluent_conv'
 // or
 const fluent = require('fluent_conv')
 
-fluent.ftl2js(str, (err, res) => {})
+fluent.ftl2js(str, (err, res) => {}, { ignoreComments: false })
```

Comment:
Are there any objections to default ignoreComments = true? or change the variable name to respectComments = false?

Reply:
Hm... I wanted to preserve the current behaviour when the comments are extracted.

Changing the default value of `ignoreComments` to `true` will require a major version bump because someone can alrealy rely on these comments in the outputed json.

Reply:
It's ok, just wanted to know 😉

Reply:
But I like the `respectComments`option with `true` as default value. Sounds more natural.

Will change it tomorrow 👍

Reply:
Done!